bin/convert-themes-to-rgb.js

```diff
@@ -0,0 +1,35 @@
+const fs = require("fs");
+const path = require("path");
+const tinycolor = require("tinycolor2");
+const colorsWithAlpha = ["toolbar", "toolbar_field"];
+const themesPath = path.join(__dirname, "..", "src", "preset-themes");
+
+const makeTinycolor = colorIn => {
```

Comment:
Copied over the tweak to alpha conversion I was missing from the themes library. That should fix issues with alpha values